charon/src/translate/translate_ctx.rs

```diff
@@ -452,6 +452,10 @@ impl<'tcx, 'ctx> TranslateCtx<'tcx, 'ctx> {
             attributes: self.translate_attributes_from_rid(def_id),
             inline: self.translate_inline_from_rid(def_id),
             public,
+            opaque: self
+                .translate_attributes_from_rid(def_id)
```

Comment:
Please compute `translate_attributes_from_rid` only once; it is already computed a few lines above.